modules/microshift-olm-deploy-op-disconnected.adoc

```diff
@@ -0,0 +1,24 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-adding-OLM-Operators-to-disconnected-cluster_{context}"]
+= Adding OLM-based Operators to a disconnected cluster
+
+For {microshift-short} clusters that are installed on restricted networks, also known as _disconnected clusters_, Operator Lifecycle Manager (OLM) by default cannot access sources hosted on remote registries because those remote sources require full internet connectivity.
+
+You can enable your cluster to use OLM-based Operators in a restricted network. The following steps are required to use OLM-based Operators in restricted networks:
+
+* Include OLM in your container image list for your mirror registry. See xref:../microshift_install/microshift-deploy-with-mirror-registry.adoc#microshift-get-mirror-reg-container-image-list_microshift-deploy-with-mirror-registry[Getting the mirror registry container image list].
+//Q:Or are we referring to link:https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images[Mirroring images for a disconnected installation] but swapping out for oc-mirror?
+//or adding a different link such as: link:https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog_installing-mirroring-installation-images[Mirroring Operator catalogs for use with disconnected clusters]
```

Comment:
Are talking here about mirroring images for the OLM itself or catalog and operators?

Just had a brief look at `oc mirror` and I'm not sure it's fully suitable for core MicroShift images right now: it can mirror images of the OpenShift, but it mirrors all of them (35GB for 4.14), so that's a bit excessive.
For mirroring operators/catalogs it's more useful and it's not limited to OCP marketplace/catalog, but can be used for other catalogs like OperatorHub.io.

For catalog and operators: best to reuse OCP's procedure, though I see it's using `oc adm catalog`, not `oc mirror`

Reply:
Whatever is necessary. I assume we treat all of the elements like we would MicroShift itself? It seems like this is not decided.
We are not to use `oc adm catalog` per @kevinrizza, so we need to come up with an example for MicroShift users.

Reply:
We should talk to the `oc-mirror` folks. You don't actually need to use the parts of oc-mirror that mirror the payload in order to mirror the olm catalogs. They're different commands.